webapp/src/components/pages/stream/Stream.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../../../../assets/scss/components/general/streaming/stream/stream.scss';
+import { useParams } from 'react-router-dom';
+
+function Stream() {
+  const { movieId } = useParams();
+
+  return (
+    <div className="stream-container">
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <video src={`http://localhost:8080/api/stream/video/${movieId}`} width="50%" controls autoPlay>
```

Comment:
We never use the back-end url: http://localhost:8080/api, use the front-end url, or a relative url `api/stream/video/${movieId}`